tests/client/amo/components/TestAddonMoreInfo.js

```diff
@@ -108,7 +108,23 @@ describe('<AddonMoreInfo />', () => {
       'Read the privacy policy for this add-on');
     // TODO: Change this to an internal `<Link>` tag and use `assert.equal`
     // once https://github.com/mozilla/addons-frontend/issues/1828 is fixed.
-    assert.include(root.privacyPolicyLink.href,
+    assert.include(root.privacyPolicyLink.props.href,
       '/addon/chill-out/privacy/');
   });
+
+  it('renders the EULA and link', () => {
+    const root = render();
```

Comment:
Since this test requires `eula` data I think it's helpful to explicitly set up the data you are depending on, like:
````js
const root = render({ 
  addon: {
    ...fakeAddon,
    has_eula: true,
  },
});
````